src/winstanley/Wdl.flex

```diff
@@ -80,7 +82,10 @@ PARAMETER_META="parameter_meta"
 META="meta"
 OBJECT="object"
 IDENTIFIER=[:letter:]([:letter:]|[:digit:]|\_)*
-SEP="sep"
+ATTR_SEP="sep"
+ATTR_DEFAULT="default"
+ATTR_TRUE="true="|"true ="
+ATTR_FALSE="false="|"false ="
```

Comment:
the whitespace can't be regexed?

Reply:
Since this is in the tokenisation step, I'm not sure it can. The lexer can read forward a few characters to detect the difference between an `ATTR_TRUE` and a `TRUE` but I'm not sure it can do regexes.

I'd be very happy to be proven wrong though!